### DISCUSSION TITLE
gif images are not playing but displayed as stationary images

### DISCUSSION THREAD

**Discussion by vivekragunathan** [thuliteio/doks#597] [Q&A]
For example, look at this [page](https://qwikref.netlify.app/docs/bash/git/#turbocharge-git). The image is a gif but does not play. It is displayed as a stationary image. It appears that during the build process, gifs are processed as regular images (converted to different sizes)

**Comment by h-enk** [comment]
In `./config/_default/params.toml`, make sure to set `smallLimit` to a value equal to or greather than the width of the gif, e.g.: ``smallLimit = "480"``

![2022-01-04_11-25-42](https://user-images.githubusercontent.com/3902872/148045161-782e66e9-32e8-4ddd-bb39-d9c53b9d8128.gif)
.

**Reply by Orilcious** [reply]
There is no ./config/_default/params.toml

There is config/_default/hugo.toml and config/_default/params.yml

No matter where I put "smallLimit", my gif still get converted to webp.

In the log I see:

WARN  Hugo Responsive Images:
Source image /blog/watchguard-grundlagen/010.gif (824x620px) on page /blog/watchguard-grundlagen/ is not wide enough to resize to 1024px wide. Resized to 824px wide

**Reply by h-enk** [reply]
With Doks v1, use e.g. 

- `img` shortcode:

```md
{{< img src="giphy.gif" width="263" height="210" alt="Alternate text" class="img-fluid blur-up d-block mx-auto">}}
```

- Render hook:

```md
---
title: "Example Post"
description: ""
summary: ""
date: 2023-09-07T16:27:22+02:00
lastmod: 2023-09-07T16:27:22+02:00
draft: false
weight: 50
images: []
categories: []
tags: []
contributors: []
pinned: false
homepage: false
resources:
- src: giphy.gif
  params:
    image:
      output: img
      alt: "Alternate text"
      title: "Title text"
      width: 263
      class: "img-fluid blur-up d-block mx-auto"
---

![](giphy.gif)

```


Documentation is in the making, for now, see https://github.com/future-wd/hugo-responsive-images. The warning is triggered incorrectly, please ignore it (will be fixed).

**Reply by Orilcious** [reply]
Thanks for the help.

Using the shortcode did not work, however adding 

```
resources:
- src: giphy.gif
  params:
    image:
      output: img
      alt: "Alternate text"
      title: "Title text"
      width: 263
      class: "img-fluid blur-up d-block mx-auto"
 ```
 
 To the header did the trick. Thanks a lot.
